decidim-core/app/controllers/decidim/devise/sessions_controller.rb

```diff
@@ -8,7 +8,8 @@ class SessionsController < ::Devise::SessionsController
 
       before_action :check_sign_in_enabled, only: :create
 
-      def create
+      def destroy
```

Comment:
I'm not sure what was the point of the `create` action but any reason to delete it?

Reply:
It was just `super`ing so it's safe to delete, right?

Reply:
I guess so